File: src/seed/files/clean-example.ts
interface UserProfile {
  id: string;
  name: string;
  email: string;
  createdAt: Date;
}

export function createUserProfile(
  name: string,
  email: string
): UserProfile {
  return {
    id: crypto.randomUUID(),
    name: name.trim(),
    email: email.toLowerCase().trim(),
    createdAt: new Date(),
  };
}

export function isValidEmail(email: string): boolean {
  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
  return emailRegex.test(email);
}

export function formatUserDisplay(user: UserProfile): string {
  const dateStr = user.createdAt.toISOString().split('T')[0];
  return `${user.name} (${user.email}) - joined ${dateStr}`;
}

File: src/seed/files/clean-example.py
from dataclasses import dataclass
from typing import Optional
from datetime import datetime


@dataclass
class UserProfile:
    name: str
    email: str
    created_at: datetime

    def display_name(self) -> str:
        return f"{self.name} ({self.email})"


def create_user(name: str, email: str) -> UserProfile:
    """Create a new user profile with validation."""
    if not name or not email:
        raise ValueError("Name and email are required")

    return UserProfile(
        name=name.strip(),
        email=email.lower().strip(),
        created_at=datetime.now(),
    )


def is_valid_email(email: str) -> bool:
    """Basic email validation."""
    return "@" in email and "." in email.split("@")[-1]
